bcl2fastq/lib/bcl2fastq_utils.py

```diff
@@ -262,47 +262,29 @@ def construct_command(self):
     def symlink_output_to_unaligned(self):
         """
         Create a symlink from `runfolder/Unaligned` to what has been defined as the output directory.
-        :return: None
         :raises: OSError if there was any problem creating the symlink, except for that it was already
                          there, in which case, do nothing.
         """
+        link_path = self.config.runfolder_input + "/Unaligned"
+        link_target_path = self.config.output
+
         try:
-            os.symlink(self.config.output, self.config.runfolder_input + "/Unaligned")
-            return None
+            log.debug("Create symlink from {} to {}.".
+                      format(link_path,
+                             link_target_path))
+            os.symlink(link_target_path, link_path)
         except OSError as e:
             if e.errno == os.errno.EEXIST:
-                log.warning("Symlink from {} to {} already exits, will not recreate it...".
-                            format(self.config.runfolder_input + "/Unaligned",
-                                   self.config.output))
+                log.warning("Symlink from {} to {} already exits, will remove it and recreate it...".
+                            format(link_path, link_target_path))
+                log.warning("Removing link: {}".format(link_path))
```

Comment:
I did it that way because it makes the log easier to read if that ends up on two lines...